lib/fetch/index.js

```diff
@@ -1990,4 +1991,72 @@ function httpNetworkFetch (
   })
 }
 
+async function fileFetch (fetchParams) {
+  const { request } = fetchParams
+  const url = requestCurrentURL(request)
+  const context = this
+  if (request.method !== 'GET') {
+    return makeNetworkError(`Fetching files only supports the GET method. Received ${request.method}`)
```

Comment:
What do browsers do with HEAD?

Reply:
I looked at what Deno does, I event kept the message the same.